### ISSUE TITLE
XSS fixes in library/custom_template/

### ISSUE THREAD

**Issue by bradymiller** [openemr/openemr#2157]
Clean up XSS prevention in scripts in library/custom_template/ directory:
https://github.com/openemr/openemr/tree/master/library/custom_template
(part of this issue: https://github.com/openemr/openemr/issues/2145)

**Comment by bradymiller** [comment]
Issue reserved by @flockclock

**Comment by danehrlich1** [comment]
Looks like @flockclock outright deleted their Github. I believe this issue is now open.

**Comment by danehrlich1** [comment]
Good additional instructions: https://github.com/openemr/openemr/issues/2169#issuecomment-460129695

**Comment by bradymiller** [comment]
Then it goes to:
@joshuaaguilar20

**Comment by joshuaaguilar20** [comment]
@bradymiller  @danehrlich1 I am commited a few updates. I just needed to confirm that some of the ajax request are not sending html I need to append to the DOM? I tried testing it with postman, but not great with php, so having a hard time finding the server code sending the response object?. I also cleaned up a little syntax. also I noticed on travis CI we are building 7.2 twice and was just curious why that was ? 


![doublebuild](https://user-images.githubusercontent.com/31646651/52244549-f73fca80-2892-11e9-9898-bc63ded7d651.png)
